view/frontend/templates/js/replacejs.phtml

```diff
@@ -800,6 +794,8 @@ ob_start();
                             allowAutoOpen = false;
                         }
                     });
+            // prefetch Shipping and Tax for multi-step checkout
+            prefetchShipping();
```

Comment:
This makes prefetch happen unnecessarily often.
consider bundling this with with the `boltcart` customer section, or
create a new `bolthints` customer section have it invalidate on `/V1/carts/`*/estimate-shipping-methods` and `/V1/guest-carts/*/estimate-shipping-methods`

Reply:
Good catch Leon! I fixed this by changing "POST" to "GET" in prefetchShipping function. We no need to invalidate any sections when we do this prefetch call.

And this is an interesting idea to handle prefetchShipping using sections down the road, let me think about it.

Reply:
sounds good.